.travis.yml

```diff
@@ -44,10 +43,10 @@ deploy:
   - provider: script
     skip_cleanup: true
     on:
-      branch: 1.1-stable
+      branch: 1.2-stable
```

Comment:
I'd leave this change out - this will be addressed separately to make the docs deploy section version agnostic.